README.md

```diff
@@ -82,13 +82,20 @@ Required parameters
 Optional parameters
 -------------------
 
+All options supported in the [Notifications API specification](https://notifications.spec.whatwg.org/#dictdef-notificationoptions):
+
 * `body`: (string) - notification message body
 * `icon`: (string) - path for icon to display in notification
 * `tag`: (string) - unique identifier to stop duplicate notifications
 * `lang`: (string) - BCP 47 language tag for the notification (default: `en`)
 * `timeout`: (integer) - number of seconds to close the notification automatically
 * `silent`: (boolean) - indicates that no sounds or vibrations should be made (default: `false`)
 * `requireInteraction`: (boolean) - keep the notification open indefinitely
```

Comment:
I think we can probably remove doc references to options that are specific to the API itself now?